content/en/posts/js/introduction-a-reactjs/index.md

```diff
@@ -0,0 +1,176 @@
+---
+date: "2014-06-04"
+route: /posts/js/introduction-a-reactjs/
+title: Introducing ReactJS
+tags:
+  - javascript
+  - reactjs
+authors:
+  - bloodyowl
+  - skinnyfoetusboy
```

Comment:
I would think it's a pretty good idea. 
I only added myself as an author for lack of a better place.